gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -37,7 +37,7 @@
 #define NNSTREAMER_PREFIX_DECODER	"libnnstreamer_decoder_"
 #define NNSTREAMER_PREFIX_FILTER	"libnnstreamer_filter_"
 #define NNSTREAMER_PREFIX_CUSTOMFILTERS	""
-#define NNSTREAMER_PREFIX_CONVERTER	""
+#define NNSTREAMER_PREFIX_CONVERTER	"libnnstreamer_converter_"
```

Comment:
Why do you need this prefix? You do not use filenames to extract subplugin names, right?

Reply:
LGTM other than this.

Reply:
It is necessary to parse the sub-plugin name from the lib files even with the `NNS_SEARCH_GETALL` option.
`_get_filenames` function requires this prefix.

Reply:
Uh... does ```_get_filenames``` really need it?

```g_str_has_prefix (string, "")``` returns TRUE
and
```strlen ("")``` returns 0
Thus, it appears that ```_get_filenames()``` will work properly with ```""``` prefixes.

Reply:
My explanation was insufficient.
The functions that you mentioned can work well with "".
But when a sub-plugin is added, `findExternalConverter` needs sub-plugin name from config info to register it.
Would it be better to move this change to PR #2382?

Reply:
Now I understand. I will modify it according to the direction of you reviewed.

Reply:
`findExternalConverter` is changed to get sub-plugins according to the mime type regardless of the name of the library.